site/pages/create-disruption.page.tsx

```diff
@@ -1,7 +1,8 @@
 import { NextPageContext } from "next";
 import { parseCookies } from "nookies";
 import { ReactElement, useState } from "react";
-import { inspect } from "util";
+import { z } from "zod";
```

Comment:
When you run this locally and submit the page with no inputs, no errors are shown for the end date or end time, even though the checkbox was not checked. Is this intended?

Reply:
Also, a few of the inputs arent clearing their errors when they are filled in after this - unsure if this was intended.

Reply:
It looks like this might be because zod only runs refinements if the initial schema is valid, so it isn't getting to that check, we can either leave it as and potentially have users receive error messages twice or we could make the initial schema a partial and have a refinement check which tests to make sure all the required fields are there

Reply:
I'll have a look at the errors that aren't being cleared